db/config.cc

```diff
@@ -1245,7 +1245,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , minimum_replication_factor_warn_threshold(this, "minimum_replication_factor_warn_threshold", liveness::LiveUpdate, value_status::Used,  3, "")
     , maximum_replication_factor_warn_threshold(this, "maximum_replication_factor_warn_threshold", liveness::LiveUpdate, value_status::Used, -1, "")
     , maximum_replication_factor_fail_threshold(this, "maximum_replication_factor_fail_threshold", liveness::LiveUpdate, value_status::Used, -1, "")
-    , tablets_initial_scale_factor(this, "tablets_initial_scale_factor", value_status::Used, 1, "Calculated initial tablets are multiplied by this number")
+    , tablets_initial_scale_factor(this, "tablets_initial_scale_factor", value_status::Used, 10, "Calculated initial tablets are multiplied by this number")
```

Comment:
We will have to hint system_tracing tables, or they will eat much of the budget.

Reply:
Yes. 

system_tracing doesn't use tablets yet.